### ISSUE TITLE
Low Latency

### ISSUE THREAD

**Issue by KevinCJohns** [unifiedstreaming/fmp4-ingest#10]
Given general industry interest in Low Latency Streaming, the Ingest Spec should touch on this. In particular how the encoder transfers data to the Media Processing Entity and how the transfer method impacts latency should be discussed. For example, the encoder should not wait till it has enough content for a full media segment (e.g., 10 seconds) before sending its data; rather it should start sending the media immediately so as to minimize latency.

**Comment by unifiedstreaming** [comment]
yes this is important , but with DASH ingest you can just use 1 second segments if you want to use low latency, only in HLS segments are often this long. In DASH-IF CMAF chunks are considered, but i don't think we have to narrow down to CMAF at this time. I can add a line that 1 second segments can be used for low latency would that be ok ?

**Comment by KevinCJohns** [comment]
The use of 1 second segments is more on the player side. I was trying to think of how the ingest process impacts latency. For example, if the customer wants 1 second segments, we probably don't want the encoder to queue up the full segment before sending to the media processing entity. Rather it should start streaming the data as soon as it is ready. This shaves some time off the end to end delay. There may be other ideas as to how to get the content to the media processing entity faster...

**Comment by unifiedstreaming** [comment]
OK, we can keep the chunked transfers in the spec, this should help with low latency aswell, this will also help with backward compatibility with the azure ingest spec which will be nice to have also from a practical perspective
